public/app/features/explore/RawList.tsx

```diff
@@ -0,0 +1,92 @@
+import { css } from '@emotion/css';
```

Comment:
could you move all the files related to this component into a separate directory ( a la `TraceView` in this very same directory) so we avoid too much clutter?

Reply:
Good call, this has been pushed up!